src/udfs/pytorch_abstract_udf.py

```diff
@@ -38,8 +38,8 @@ def __init__(self):
         AbstractClassifierUDF.__init__(self)
         nn.Module.__init__(self)
 
-    def get_device(self):
-        return next(self.parameters()).device
+    #def get_device(self):
```

Comment:
Are these changes required? I remember we decided to revert these back. Please double-check, thanks!

Reply:
Yes, I think these changes are required. We are storing the device object using the assign_device() function in the GPU Compatible class as one of its variables. So we can directly access that object instead of using a get_device() function. I believe that was the reason we decided to eliminate this functionality. What do you think?